mocks/hawthorn/student/models.py

```diff
@@ -74,6 +86,41 @@ def has_profile_image(self):
 
 
 class CourseEnrollmentManager(models.Manager):
+
+    def num_enrolled_in_exclude_admins(self, course_id):
+        """
+        Returns the count of active enrollments in a course excluding instructors, staff and CCX coaches.
+
+        Arguments:
+            course_id (CourseLocator): course_id to return enrollments (count).
+
+        Returns:
+            int: Count of enrollments excluding staff, instructors and CCX coaches.
+
+        """
+        # To avoid circular imports.
+        from student.roles import CourseCcxCoachRole, CourseInstructorRole, CourseStaffRole
+        course_locator = course_id
+
+        # This will break, but leaving for completeness until we need to remove it
+        # TODO Raise exception if we get this
+        if getattr(course_id, 'ccx', None):
+            raise Exception('CCX is not supported')
+
+        staff = CourseStaffRole(course_locator).users_with_role()
+        admins = CourseInstructorRole(course_locator).users_with_role()
+        coaches = CourseCcxCoachRole(course_locator).users_with_role()
+        # import pdb; pdb.set_trace()
+        qs = super(CourseEnrollmentManager, self).get_queryset()
```

Comment:
I'll change this and remove the remarked out `set_trace`

Reply:
Actually, I'm going to leave it because it is calling a different method in super. I also need to fix this mock method because I realize I left it incomplete. Will remove the `set_trace` and remarked out code after updating the `super get_queryset call`